### PR TITLE
Persson Tine Project-business site

### PR DESCRIPTION
**Pull request by gtine**

### CODE REVIEW

**Comment by gtine** [comment]
Creating a new since some stuff got lost in last version.

File: code/index.html [hunk 1]
```diff
@@ -4,27 +4,77 @@
...
+    <div class="both"> 
+    <section class="faq-section">
+      <h1 class="faq-topic">FAQS</h1>
+    
+    <div class="accordion">
+      <h2 class="header1" id="header1"> 
+        If I want to visit the café, do I need to buy an entrance ticket?</h2>
+        <p class= "content1" id="content1">
+          Yes, you will need to buy an entrance ticket to visit the café.</p>
+    </div>
```
Comment (HIPPIEKICK) [inline code/index.html:28]: Check indentation! And there should be no space between = and value:
`<p class="content1" id="content1">`

File: code/index.html [hunk 2]
```diff
@@ -4,27 +4,77 @@
...
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <div class="form-container">
+    <form action="https://httpbin.org/anything" method="post">
+      <h1 class="form-topic">Get the latest Updates - <br> Subscribe below!</h1>
+      <input name="Name" type="text" placeholder="Name">
+      <input name="E-mail" type="text" placeholder="Email">
+      <input name="Number" type="Phone-number" placeholder="Phone-number">
```
Comment (HIPPIEKICK) [inline code/index.html:57]: Don't forget the labels!

File: code/index.html [hunk 3]
```diff
@@ -4,27 +4,77 @@
...
+
+    <div class="terms">
+      <p>Accept terms and conditions</p>
+      <input type="checkbox">
+    </div>
```
Comment (HIPPIEKICK) [inline code/index.html:62]: Label 👀

File: code/index.html [hunk 4]
```diff
@@ -4,27 +4,77 @@
...
+
+      <label for="Male">Male</label>
+      <input type="radio">
+
+      <label for="Female">Female</label>
+      <input type="radio">
+
+      <label for="Genderqueer/Non-Binary">Genderqueer/Non-Binary</label>
+      <input type="radio" >
```
Comment (HIPPIEKICK) [inline code/index.html:71]: Radio buttons need the `name` attribute in order to work together as a group. Read more in the general feedback document. Also, when you have the label outside of the input, you need to add a corresponding `id` to the input

File: code/style.css [hunk 5]
```diff
@@ -2,6 +2,175 @@
...
-.active + .answer {
-  /* some styling here */
+/* Hero Section */
+
+body {
+  background-image: url("./images/garden.jpg");
+  margin:0px;
+  object-fit: cover;
+  background-attachment: fixed;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:12]: Nice!

File: code/style.css [hunk 6]
```diff
@@ -2,6 +2,175 @@
...
+
+.tenko {
+  position: absolute;
+  top:0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background-color: rgba(255, 255, 0, 0.5);
+  transform: rotate(90deg);
```
Comment (HIPPIEKICK) [inline code/style.css:32]: Looks like this is making your content to wide (i.e. not responsive):
Skärmavbild 2021-02-04 kl  18 14 32

File: code/style.css [hunk 7]
```diff
@@ -2,6 +2,175 @@
...
+.accordion {
+  border: 3px solid;
+  margin: 15px;
+  padding: 20px;
+  text-align: center;
+  background: rgb(252, 214, 221);
+}
+
+
+.content1, .content2, .content3, .content4 {
```
Comment (HIPPIEKICK) [inline code/style.css:73]: Maybe they can have the same class name? :)

File: code/style.css [hunk 8]
```diff
@@ -2,6 +2,175 @@
...
+
+
+
+
+
+
+    
+  
+
+
```
Comment (HIPPIEKICK) [inline code/style.css:175]: One empty line is enough ;)

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Tine! Some small changes need to be done to this project. Take a look at my comments and the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) for more tips.

**Comment by gtine** [comment]
I have made all the changes that was requested. Ive had some trouble understanding hos to add changes in the right file but I hope Ive done it the right way now (first cloning my file then pushing back (not to origin master)? Please let me know otherwise or if there is another way. Thanks for your feedback! Tine

**Pull request closed without merging**